db/migrate/20170613144708_add_sti_to_container_image.rb

```diff
@@ -0,0 +1,20 @@
+class AddStiToContainerImage < ActiveRecord::Migration[5.0]
+  class ContainerImage < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    add_column :container_images, :type, :string
```

Comment:
I would create the first migration for structural change:
```ruby
def change 
  add_column :container_images, :type, :string
end
```
and the other with your data migration.

the reason can up method was not called that your db is probably already migrated.
try rollback rake `db:rollback STEP=5` (5 means 5 migrations back ) 
Does this work for you ?

**Why separate?**
these are two [different things](https://www.quora.com/What-are-the-differences-between-DDL-and-DML). It can be also helpfull for debbuging problems.
You also create more accurate names of migration. something like `AddTypeToContainerImage` and  `SetTypeTo`...

thanks